service/models.py

```diff
@@ -157,6 +157,15 @@ def find_by_all(cls, wishlist_id=None, name=None, customer_id=None):
 
         return cls.query.filter(*queries)
 
+    @classmethod
+    def remove_all(cls):
```

Comment:
Since we're not using the endpoint, can we remove it?

Reply:
Se above comment